crates/smb-transport/scripts/ksmbd.sh

```
@@ -1,4 +1,4 @@
-#!/bin/bash
+git push --set-upstream origin 161-add-unchecked-dfs-root#!/bin/bash
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)